docs/reference/master-configuration-reference.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Master config reference"
+layout: default
+sort: 4
+---
+
+# Configuration file reference of nfd-master
+{: .no_toc}
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+---
+
+See the
+[sample configuration file](https://github.com/kubernetes-sigs/node-feature-discovery/blob/{{site.release}}/deployment/components/master-config/nfd-master.conf.example)
+for a full example configuration.
+
+## noPublish
+
+`noPublish` option disables updates to the Node objects in the Kubernetes
+API server, making a "dry-run" flag for nfd-master. No Labels, Annotations or
+ExtendedResources of nodes are updated.
+
+Default: `false`
+
+Example:
+
+```yaml
+noPublish: true
+```
+
+## extraLabelNs
+`extraLabelNs` specifies a comma-separated list of allowed feature
+label namespaces. This option can be used to allow
+other vendor or application specific namespaces for custom labels from the
+local and custom feature sources, even though these labels were denied using
+the `denyLabelNs` parameter.
+
+The same namespace control and this option applies to Extended Resources (created
+with `resourceLabels`), too.
+
+Default: *empty*
+
+Example:
+
+```yaml
+extraLabelNs: {"added.ns.io","added.kubernets.io"}
+```
+
+## denyLabelNs
+`denyLabelNs` specifies an array list of excluded
```

Comment:
Just an array, or list. In yaml they are called array but would list be more descriptive for the user. WDYT?

```suggestion
`denyLabelNs` specifies a list of excluded
```